src/main/resources/application-local.yml

```diff
@@ -16,3 +16,6 @@ spring:
 
 court-case-service:
   base-url: http://localhost:8082/
+
```

Comment:
For the line above (actually my fault this one) but I changed that port and committed by mistake. Basically the gateway wants to run on 8080. I never should have committed it.

Reply:
Ah right, I never actually ended up running with these settings anyway so I'll remove both as I don't think we have a well defined use case for them right now